### DISCUSSION TITLE
在原神里荧的哥哥叫啥

### DISCUSSION THREAD

**Discussion by zgyhim** [share121/inter-knot#5799] [公告]
在原神里荧的哥哥叫啥?

🤔

**Comment by jin101018** [comment]
“空”吧

**Comment by zgyhim** [comment]
空间是一格半 谁也出不去
几寸也不满意
思想是循环 温饱思淫欲
真理就是没真理
在成人之前 真想先成为自己
在世界毁灭之前 真想先毁灭自己
我向我看齐 于是一直往下去
没人在看你
万物皆有灵 只是我太空虚
在成人之前 真想先成为自己
在世界毁灭之前 真想先毁灭自己
真理就是没真理
那谁想听 那谁想听
<img src="https://bbs-static.miyoushe.com/static/2024/08/07/8fb21eb68601d469ff4eb2af32c9b869_2290071814068511242.png" alt="流浪者-在意" width="75" >

**Comment by hsd221** [comment]
首先排除空喵
